nodejs/week1/app.js

```diff
@@ -0,0 +1,8 @@
+// Warmup Exercice 
+// parse command line arguments
+const listNumber = process.argv.slice(2).map(num => Number(num));
+let sum = 0;
+const average = sum / listNumber.length;
```

Comment:
I doubt if this will work. Sum is never reassigned and cinse sum =0, average will always be 0.

Reply:
Thank you for let me I have update wrong file in app.js in git

Reply:
I have updated code at app.js file.